bin/test_pyodide.mjs

```diff
@@ -12,20 +12,33 @@ for (const fileName of fileNames) {
 
 const pyodide = await pyodide_pkg.loadPyodide()
 await pyodide.loadPackage([
-    'mpmath',  // provided by pyodide
-    'numpy',  // built by pyodide
-    `../dist/${sympy}`  // git version sympy
+    'mpmath',  // provided by Pyodide
+    'numpy',  // built by Pyodide
+    'pytest',  // provided by Pyodide
+    `../dist/${sympy}`  // git version SymPy
 ])
-
-let split = 'None'
-if (argv[2]) {
-    if (argv[2].startsWith('--split=')) {
-        split = `'${argv[2].slice(8)}'`
-    }
-}
+console.log('Packages loaded successfully')
 
 pyodide.runPython(`
+print('Pyodide Python process started')
+
+import os
+import sys
+print('sys & os imported successfully')
+
 import sympy
-if not sympy.test(split=${split}, subprocess=False):
-    exit(1)
+print('sympy import successfully')
+
+print(os.getcwd())
+
+print(\n'CWD:\n----')
```

Comment:
This is giving a syntax error:
```
Python exception:
Traceback (most recent call last):
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 421, in eval_code
    CodeRunner(
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line 237, in __init__
    self.ast = next(self._gen)
  File "/lib/python3.10/site-packages/_pyodide/_base.py", line [14](https://github.com/sympy/sympy/actions/runs/3547377518/jobs/5957431774#step:6:15)1, in _parse_and_compile_gen
    mod = compile(source, filename, mode, flags | ast.PyCF_ONLY_AST)
  File "<exec>", line 14
    'CWD:
    ^
SyntaxError: unterminated string literal (detected at line 14)
```
I'm not sure how JS handles this but obviously this is a Python code string literal inside a js file. Running this in node gives:
```js
> console.log(`
... print('Pyodide Python process started')
... import os
... import sys
... print('sys & os imported successfully')
... import sympy
... print('sympy import successfully')
... print(os.getcwd())
... print(\n'CWD:\n----')
... for iteration in os.walk(os.getcwd()):
...     print(iteration)
... print(\n'SymPy:\n------')
... for iteration in os.walk('/lib/python3.10/site-packages'):
...     print(iteration)
... exit_code = sympy.test(subprocess=False)
... sys.exit(exit_code)
... `)

print('Pyodide Python process started')
import os
import sys
print('sys & os imported successfully')
import sympy
print('sympy import successfully')
print(os.getcwd())
print(
'CWD:
----')
for iteration in os.walk(os.getcwd()):
    print(iteration)
print(
'SymPy:
------')
for iteration in os.walk('/lib/python3.10/site-packages'):
    print(iteration)
exit_code = sympy.test(subprocess=False)
sys.exit(exit_code)
```
So it looks like the `\n` are being interpreted as newlines in the JS string before it gets executed by Python.